Deprecate packagemanifests commands (#4838)
operator-framework/operator-sdk: website/content/en/docs/olm-integration/cli-overview.md

### Package Manifests

> The operator-framework is migrated towards using bundles by default, hence `generate packagemanifests` and `run packagemanifests` commands have been deprecated. For more

Reviewer: ```suggestion
The operator-framework is removing support for the packagemanifests format in the near future, and migration efforts are currently underway. Therefore `generate packagemanifests` and `run packagemanifests` commands have been deprecated. For more
```

> details on bundle format refer [here][bundle]. In order to migrate package manifests to 

Reviewer: ```suggestion
details on bundle format refer [here][bundle]. To migrate from packagemanifests to the
```

> bundle format, you can use [`operator-sdk pkgman-to-bundle`][cli-pkgman-to-bundle] command.

Reviewer: ```suggestion
bundle format, use the [`operator-sdk pkgman-to-bundle`][cli-pkgman-to-bundle] command.
```
